### PR TITLE
abstraction level, cell type, species, brain region, author, organization values can be lists

### PR DESCRIPTION
**Pull request by aopy**

### CODE REVIEW

File: client/nar/brainsimulation.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ class ModelProject(KGObject):
         "dateCreated": "schema:dateCreated",
     }
 
-    def __init__(self, name, owner, author, description, date_created, private, collab_id, alias=None,
```
Comment (apdavison) [inline client/nar/brainsimulation.py]: should be `authors`, not `people`

File: client/nar/brainsimulation.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def __init__(self, name, owner, author, description, date_created, private, coll
         self.organization = organization
         self.abstraction_level = abstraction_level
         self.private = private
-        self.author = author
```
Comment (apdavison) [inline client/nar/brainsimulation.py]: as above

File: client/nar/brainsimulation.py [hunk 3]
```diff
@@ -109,24 +115,44 @@ def save(self, client, exists_ok=True):
...
+             ]
         if self.PLA_components is not None:
             data["PLAComponents"] = self.PLA_components
         if self.alias is not None:
             data["alias"] = self.alias
         if self.model_of is not None:
             data["modelOf"] = self.model_of
         if self.brain_region is not None:
-            data["brainRegion"] = self.brain_region.to_jsonld()
+            data["brainRegion"] = [
```
Comment (apdavison) [inline client/nar/brainsimulation.py]: what is the reason for this change?

Comment (HFragnaud) [reply client/nar/brainsimulation.py]: for now it is not mandatory for the data migration but we will offer the possibility to add several brains regions, species etc... to a model project. I thought it would be better to add the possibility to save all the parameters as a list now, so we will not have to change the code after.

Comment (apdavison) [reply client/nar/brainsimulation.py]: ok, but then it should be `data["brainRegion"] = [br.to_jsonld() for br in self.brain_regions]` - i.e. keep the serialization in the BrainRegion class, within the `to_jsonld()` method. This ensures we don't have different serializations in different places.

Comment (apdavison) [reply client/nar/brainsimulation.py]: Just waiting for this to be fixed, then I can merge.
